app/src/main/java/co/temy/android/ktx/crypto/CipherExtensions.kt

```diff
@@ -0,0 +1,98 @@
+package co.temy.android.ktx.crypto
```

Comment:
Would be nice to have some sample documentation, maybe with another pull request in future we can provide them